dev/com.ibm.websphere.appserver.features/visibility/public/mpLRA-1.0/io.openliberty.mpLRA-1.0.feature

```diff
@@ -11,7 +11,8 @@ IBM-API-Package: \
   org.eclipse.microprofile.lra.annotation; type="stable", \
   org.eclipse.microprofile.lra.annotation.ws.rs; type="stable";
 -features=io.openliberty.org.eclipse.microprofile.lra-1.0, \
-          com.ibm.websphere.appserver.jaxrs-2.1
+          com.ibm.websphere.appserver.jaxrs-2.1, \
+          io.openliberty.mpCompatible-1.0; ibm.tolerates:=4.0
```

Comment:
It is important to ensure LRA working with other MP features because this is the only one LRA feature. This is a good tolerance.